Source/BSGOutOfMemoryWatchdog.m

```diff
@@ -214,7 +214,7 @@ - (NSMutableDictionary *)generateCacheInfoWithConfig:(BugsnagConfiguration *)con
     app[@"id"] = systemInfo[@BSG_KSSystemField_BundleID] ?: @"";
     app[@"name"] = systemInfo[@BSG_KSSystemField_BundleName] ?: @"";
     app[@"releaseStage"] = config.releaseStage;
-    app[@"version"] = systemInfo[@BSG_KSSystemField_BundleVersion] ?: @"";
+    app[@"version"] = systemInfo[@BSG_KSSystemField_BundleShortVersion] ?: @"";
```

Comment:
Do we want to add the bundle version in as well? Think it would something like
`app[@"bundleVersion"] = systemInfo[@BSG_KSSystemField_BundleVersion] ?: @"";`

Reply:
Yeah - having both makes sense 👍